supplementary_style_guide/glossary_terms_conventions/general_conventions/d.adoc

```diff
@@ -431,6 +477,16 @@ Red Hat Directory Server conforms to LDAP standards.
 
 *See also*:
 
+[[disk-encryption]]
+==== image:images/yes.png[yes] disk encryption (noun)
+*Description*: The encryption of data while it is stored on a disk. Also known as block device encryption or encryption at rest.
```

Comment:
```suggestion
*Description*: _Disk encryption_ of data while it is stored on a disk. Also known as block device encryption or encryption at rest.
```
- Defining _disk encryption` as `encryption of data on a disk` isn't really defining the term. Maybe something like:
 
`_Disk encryption_ is a security feature to protect access to sensitive information stored on a disk by using mathematical functions to convert disk data into a format that cannot be deciphered easily by unauthorized processes or users.`